runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -356,18 +381,30 @@ default Color insaneValueColor()
 		keyName = "insaneValuePrice",
 		name = "Insane value price",
 		description = "Configures the start price for insane value items",
-		position = 24
+		position = 26
 	)
 	default int insaneValuePrice()
 	{
 		return 10000000;
 	}
 
+	@ConfigItem(
+		keyName = "interpolateLootColors",
+		name = "Smoothed color tiers",
+		description = "Configures whether or not color is smoothly blended between tiers based on item value",
+		position = 27
+	)
+	default boolean interpolateLootColors()
+	{
+		return false;
+	}
+
 	@ConfigItem(
 		keyName = OWNERSHIP_FILTER_MODE,
 		name = "Ownership filter",
 		description = "Show all items, takeable items, or only your or your group's drops.",
-		position = 25
+		position = 25;
```

Comment:
This should not have a semicolon

Reply:
Accidental typo when I was resolving the merge conflict